_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -54,6 +56,35 @@ popular Bitcoin infrastructure software.
     problems that are the result of this bug---but developers are still
     investigating.
 
+- **US Patent 7,110,538 has expired:** Bitcoin transactions are secured using
+  [ECDSA][] (the _Elliptic Curve Digital Signature Algorithm_). Verifying
+  signatures involves multiplying points on the elliptic curve by scalars.
+  Typically, each transaction input requires one or more signature verifications,
+  meaning that syncing the Bitcoin block chain can require many millions of these
+  elliptic curve point multiplications. Any technique to make point
+  multiplications more efficient therefore has the potential to significantly
+  speed up Bitcoin's initial sync.
+
+    In 2011, [Hal Finney described][finney endomorphism] a way to more
```

Comment:
Maybe call it GLV endomorphism? It wasn't invented by Hal Finney. The secp256k1 curve was specifically designed to enable GLV; Hal pointed this out and started looking into how much of a savings it would be.